frontend/app/components/encryptable/row.js

```diff
@@ -109,22 +135,26 @@ export default class RowComponent extends Component {
   }
 
   @action
-  showPassword() {
+  showAttribute(attribute) {
     this.fetchEncryptable();
-    this.isPasswordVisible = true;
+    this[
+      `is${attribute.charAt(0).toUpperCase()}${attribute.slice(1)}Visible`
```

Comment:
what does this do?

Reply:
this[`is${capitalize(attribute)}Visible`] = true;

Basically the same as this, changed it to what I used in other Places